packages/riipen-ui/src/components/VerticalProgressBar.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import withClasses from "../utils/withClasses";
+
+import Grid from "./Grid";
+import GridItem from "./GridItem";
+import VerticalProgressBarItem from "./VerticalProgressBarItem";
+
+const VerticalProgressBar = ({ progresses }) => {
+  const renderProgress = () => {
+    return progresses.map((progress, index) => (
+      <GridItem key={`progress-${index}`}>
+        <VerticalProgressBarItem
+          progress={{ ...progress, noBar: index >= progresses.length - 1 }}
+        />
+      </GridItem>
+    ));
+  };
+  return (
+    <div>
```

Comment:
Might be able to get rid of this exterior div?

Reply:
Done 👍